src/types/events/base-events.ts

```diff
@@ -425,6 +426,40 @@ export interface FileUnsharedEvent {
   event_ts: string;
 }
 
+export interface FunctionParams {
```

Comment:
No action needed here but just calling out that in my head, eventually I'd like to see many of these types land in the `@slack/types` project.

Reply:
Totally! And I eagerly await the day that's the case. 😄